convert2rhel/data/9/x86_64/configs/rocky-9-x86_64.cfg

```diff
@@ -10,10 +10,10 @@ gpg_fingerprints = 702d426d350d275d
 excluded_pkgs =
   rocky-obsolete-packages
   rocky-gpg-keys
+  rocky-logos
```

Comment:
The swap doesn't work or this is a mistake?

Reply:
It doesn't work due to some conflicting files 🥲